src/main/webapp/components/simple-search/simple-search.js

```diff
@@ -69,6 +75,7 @@ const SimpleSearch = props => {
               setPageIndex(0)
               setQuery(query)
               onClear()
+              transformToSourceIds(query)
```

Comment:
:pencil2: This function could return a new object that has the desired structure. I think it's generally better not to mutate things if you can help it